### PR TITLE
chore: integrate husky & lintstaged & update  gitignore

### PR DESCRIPTION
**Pull request by SteveYuOWO**
image
Code is not formatted correctly by eslint. 
Adding lintstaged to automatically format code before committing

### CODE REVIEW

File: app.config.ts
Comment (dohooo) [inline app.config.ts:1]: Hmm, we just need to use the .env.common file currently. So, I think line 6 isn't necessary.

**Review by dohooo — CHANGES_REQUESTED** [review]
Thx for your contribution! Could you change the target to staging? 🤗

File: app.config.ts
Comment (SteveYuOWO) [reply app.config.ts:1]: > Thx for your contribution! Could you change the target to staging? 🤗

Thanks for your reply. Target has been changed to staging.

Comment (SteveYuOWO) [reply app.config.ts:1]: > Hmm, we just need to use the .env.common file currently. So, I think line 6 isn't necessary.

okay, `.env` configuration has been removed

**Review by dohooo — APPROVED** [review]

**Pull request merged**